dev/io.openliberty.concurrent.internal.cdi/bnd.bnd

```diff
@@ -53,7 +53,7 @@ instrument.classesExcludes: io/openliberty/concurrent/internal/cdi/resources/*.c
   com.ibm.ws.kernel.service,\
   com.ibm.ws.threading,\
   io.openliberty.jakarta.annotation.2.1,\
-  io.openliberty.jakarta.cdi.4.0,\
-  io.openliberty.jakarta.concurrency.3.0,\
+  io.openliberty.jakarta.cdi.4.1,\
```

Comment:
If this bundle is now only for EE11, should the Java version be moved up to 21 and the explicit imports be removed?